core/build.gradle

```diff
@@ -21,6 +21,7 @@ managedDependencies {
         compile "io.netty:$it"
     }
     compile 'io.netty:netty-transport-native-epoll:linux-x86_64'
+    compile 'io.netty:netty-transport-native-kqueue:osx-x86_64'
```

Comment:
We will have to make sure kqueue is not included in the generated pom.xml when releasing from OSX. I'd just modify the pom generation part of the build script.